service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -71,6 +79,15 @@ class DatasetServiceTest {
           StorageSystem.TERRA_DATA_REPO,
           metadata,
           null);
+
+  private final Dataset externalDataset =
```

Comment:
Not sure why these fields are static.. do they rely on data which changes in each test run? (looking at the change history, it's probably my fault that they weren't static before)

Reply:
Do you mean "not sure why these fields **aren't** static"? If so, should I make them all static?

Reply:
> Do you mean "not sure why these fields aren't static"? If so, should I make them all static?

Yes, that's what I meant. If you could make them all static I think that would be better.